### ISSUE TITLE
Wall of text on Card Screen...

### ISSUE THREAD

**Issue by rkaycom** [seud0nym/tch-gui-unhide#199]
![2024-06-03_133915](https://github.com/seud0nym/tch-gui-unhide/assets/143233309/cbdc5c16-a923-4ac6-bed6-f78fbc4bf872)

I'm getting this wall of text displaying on the card screen.

Technicolor TG800vac
Software Version 17.2
Firmware Version 17.2.0284-820-RA
tch-gui-unhide Version 2024.05.31@09:52

**Comment by seud0nym** [comment]
Hi.

Two questions:

1. Do you have any Dynamic DNS services defined?

1. Could you please post the out from this command:

```
transformer-cli get rpc.ddns.status
```

Thanks.

**Comment by bravohe** [comment]
> Hi.
> 
> Two questions:
> 
> 1. Do you have any Dynamic DNS services defined?

No

> 2. Could you please post the out from this command:
> 
> ```
> transformer-cli get rpc.ddns.status
> ```

rpc.ddns.status [string] = No error received from server


> 
> Thanks.

**Comment by seud0nym** [comment]
I think I have fixed the problem. You can apply the fix with the following command:
```
./tch-gui-unhide -u && ./tch-gui-unhide -y
```

**Comment by rkaycom** [comment]
Appears to be fixed. I haven't found any other issues but haven't been going out of my way to find them either. Many thanks.

**Comment by seud0nym** [comment]
Fixed in release 2024.10.10.
